### ISSUE TITLE
Training Multi-Speaker Model on vctk got bad results!

### ISSUE THREAD

**Issue by Frondewu** [r9y9/deepvoice3_pytorch#197]
Hi, r9y9, I trained 600k steps of multi-speaker model on the preprocessed vctk, but the performance is not as good as the model you released. I see that the number of training steps of your model is 300K + 300K , and the learning rate of your model is 5.7e-5. Can you explain the training process? Thanks!
